packages/nextjs-mf/src/plugins/container/runtimePlugin.ts

```diff
@@ -0,0 +1,68 @@
+import { FederationRuntimePlugin } from '@module-federation/runtime/type';
+
+export default function (): FederationRuntimePlugin {
+  return {
+    name: 'custom-plugin',
```

Comment:
Will this runtimePlugin be used in user's output ? If yes , we need to set the name more unique , because the plugin will not merge if the name is the same.

Reply:
yes ill give it a real name, thank you